overlord/snapstate/snapmgr.go

```diff
@@ -486,10 +486,30 @@ func (m *SnapManager) ensureUbuntuCoreTransition() error {
 	return nil
 }
 
+// atSeed implements at seeding policy for refreshes.
+func (m *SnapManager) atSeed() error {
+	m.state.Lock()
+	defer m.state.Unlock()
+	var seeded bool
+	err := m.state.Get("seeded", &seeded)
```

Comment:
Should we set `seeded` once the operation is complete, or is it just for the tests?

Reply:
that's done in devicestate